osu.Game.Tests/Visual/Ranking/TestSceneResultsScreen.cs

```diff
@@ -56,6 +56,17 @@ protected override void LoadComplete()
             });
         }
 
+        [Test]
+        public void TestScaling()
+        {
+            // scheduling is needed as scaling the content immediately causes the entire scene to shake badly, for some odd reason.
+            AddSliderStep("scale", 0.5f, 1.6f, 1f, v => Schedule(() =>
```

Comment:
I have no idea what's making this schedule required, but removing it would make the content process the `Scale` change for one frame, and then process the `Size` change in the frame after.

Perhaps this method is being executed at a specific point in the update thread in which this can happen.